dts/bindings/gpio/hpmicro,hpm-gpio.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 HPMicro
+# SPDX-License-Identifier: Apache-2.0
+
+description: HPMicro GPIO node
+
+compatible: "hpmicro,hpm-gpio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  hpmicro-gpio-port:
+    type: int
+    description: |
+      GPIO Port Number
+      GPIOA (0UL)
+      GPIOB (1UL)
+      GPIOC (2UL)
+      GPIOD (3UL)
+      GPIOE (4UL)
+      GPIOF (5UL)
+      GPIOX (13UL)
+      GPIOY (14UL)
+      GPIOZ (15UL)
+    required: true
```

Comment:
This property doesn't seem to be used anywhere in the GPIO driver?

Reply:
> This property doesn't seem to be used anywhere in the GPIO driver?

In gpio_hpmicro.c L237:
`		.port_base = DT_INST_PROP(n, hpmicro_gpio_port),	\`